js/src/util/vector.ts

```diff
@@ -113,6 +113,21 @@ export function createElementComparator(search: any) {
             return true;
         };
     }
+    // Compare Rows
+    if (search instanceof Row) {
+        const n = search.length;
+        const fns = [] as ((x: any) => boolean)[];
+        for (let i = -1; ++i < n;) {
+            fns[i] = createElementComparator((search as any).get(i));
+        }
+        return (value: any) => {
+            if (!(value.length === n)) { return false; }
+            for (let i = -1; ++i < n;) {
+                if (!(fns[i](value.get(i)))) { return false; }
+            }
+            return true;
+        };
+    }
```

Comment:
Note I had to create separate comparator logic for Rows and Vectors, since `search.constructor != value.constructor` in this approach.

Reply:
This is no longer an issue now that there's no inner class